docs/getting_started_guide/ISO_Build/Building_SIMP_From_Source.rst

```diff
@@ -22,60 +24,76 @@ Download the CentOS/RedHat installation media:
 Generating The ISO!
 -------------------
 
-Change into the ``simp-core`` directory.
+Clone simp-core:
 
-.. code::
+.. code-block:: bash
 
-   $ cd simp-core
+  $ git clone https://github.com/simp/simp-core
+  $ cd simp-core
 
 Check out your desired branch of SIMP:
 
 * To check out a stable SIMP release, check out a tag (*Recommended*):
 
 .. code::
 
-   $ git checkout tags/6.0.0-0
+   $ git checkout tags/6.1.0-0
 
 * To check out an unstable SIMP release, check out the latest ``master``:
 
 .. code::
 
    $ git checkout master
 
-Run ``bundle`` to make sure that all of the build tools and dependencies are installed and up to date:
+Run ``bundle`` to make sure that all of the build tools and dependencies are
+installed and up to date:
 
 .. code::
 
    $ bundle install
 
-Make sure all of the source materials that were downloaded above are in your current working directory.
+Make an ``ISO`` directory, and copy in the CentOS/RHEL installation media:
 
-Run the ``build:auto`` rake task to create a bootable ISO using the following template:
+.. code-block:: bash
 
-.. code::
+  $ mkdir ISO
+  $ cp </path/to/dvd*.iso> ISO
 
-   $ bundle exec rake build:auto[<Directory containing install media>,<SIMP version>]
+Run the ``rpm_docker`` beaker suite, toggling build options with environment
+variables:
 
-For example:
+.. code-block:: bash
 
-.. code::
+  $ <build ENV vars> bundle exec rake beaker:suites[rpm_docker,default]
 
-   $ # for SIMP 6
-   $ bundle exec rake build:auto[/path/to/ISOs,6.X]
+Build ENV vars:
 
-   $ # for SIMP 5 and CentOS 7
-   $ bundle exec rake build:auto[/path/to/ISOs,5.1.X]
+  * ``SIMP_BUILD_docs`` - (yes|no) - Toggle doc builds. Note that doc builds
+    take a long time.
 
-   $ # for SIMP 4 and CentOS 6
-   $ bundle exec rake build:auto[/path/to/ISOs,4.2.X]
+  * ``RSYNC_NO_SELINUX_DEPS`` - (yes|no) - Force the earliest version of
```

Comment:
Might want to just give a brief "if you're building against updated repositories, you need to set this"